openqa-search-maintenance-core-jobs

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+    cat << EOF
+Usage: $(basename "$0") -a SUSE:Maintenance:II:RR [-d days]
+
+The goal of this script is basically to fill the section
+"regression tests" in the log of QA test report,
+for a given update inside job groups:
+- Maintenance: Single Incidents / Core Incidents
+- Maintenance: Aggregated updates / Core Maintenance Updates
+and search for regression tests related with the update.
+Default search within the last 5 days for "Aggregated updates" builds
+
+Usage: eg
```

Comment:
```suggestion
Usage examples:
```

I would also cut this list short. Maybe just one line without and one with the `-d` parameter.

Reply:
Changed